lib/manager/jenkins/extract.ts

```diff
@@ -15,19 +72,29 @@ export function extractPackageFile(content: string): PackageFile | null {
 
     if (match) {
       const { depName, currentValue, comment } = match.groups;
-      const dep: PackageDependency = {
-        datasource: datasourceJenkins.id,
-        versioning: dockerVersioning.id,
-        depName,
-        currentValue,
-      };
-
-      if (isSkipComment(comment)) {
+      const dep = getDependency(depName, currentValue, null, null);
+
+      if (!dep.skipReason && isSkipComment(comment)) {
         dep.skipReason = SkipReason.Ignored;
       }
       deps.push(dep);
     }
   }
+  return deps;
+}
+
+export function extractPackageFile(
+  content: string,
+  fileName: string
+): PackageFile | null {
+  logger.trace('jenkins.extractPackageFile()');
+  const deps: PackageDependency[] = [];
+
+  if (/\.ya?ml$/.test(fileName)) {
```

Comment:
Create a module scope const for regex

Reply:
Add `const reYaml = /\.ya?ml$/g;` outside function
```suggestion
  if (reYaml.test(fileName)) {
```

`reYaml` can be any useful name

Reply:
Done